dll/cpl/timedate/lang/cs-CZ.rc

```diff
@@ -1,3 +1,13 @@
+/*
+ * PROJECT:     ReactOS Timedate Control Panel
+ * LICENSE:     GPL-2.0-or-later (https://spdx.org/licenses/GPL-2.0-or-later)
+ * PURPOSE:     Czech resource file
+ * TRANSLATORS: Copyright 2005 Jirka Ďoubek (Potapnik) <jirka@studioprojekt.cz>
+ *              Copyright 2007 Kamil Hornicek (Preston Pigglessworth) <tykef@atlas.cz>
```

Comment:
```suggestion
 * TRANSLATORS: Copyright 2005 Jirka Ďoubek (potapnik) <jirka@studioprojekt.cz>
 *              Copyright 2007 Kamil Hornícek <tykef@atlas.cz>
```
Kamil Hornícek's name is usually used without the nick in the headers throughout the source tree (it's not a hard requirement anyway).